shoes-core/lib/shoes/slot.rb

```diff
@@ -205,6 +209,15 @@ def position_element(_element, _current_position)
       raise 'position_element is a subclass responsibility'
     end
 
+    def position_attached_element(element)
+      attached_left = element.attached_to.element_left + element.style[:left].to_i
+      attached_top  = element.attached_to.element_top + element.style[:top].to_i
+
+      position_element_at element, attached_left, attached_top
+
+      CurrentPosition.new attached_left, attached_top, attached_top
```

Comment:
So after we position the attached element we continue after the attached element? That is... interesting. I sort of expected it to be free floating and not influence current_position at all (like normal elements with their left/top set). Is that the behavior in Shoes 3?